app/views/RoomInfoView/index.js

```diff
@@ -142,6 +148,20 @@ class RoomInfoView extends React.Component {
 		}
 	}
 
+	goRoom = async() => {
+		const { roomUser } = this.state;
+		const { username: name } = roomUser;
+		const { navigation } = this.props;
+		const result = await RocketChat.createDirectMessage(name);
+		if (result.success) {
+			await navigation.navigate('RoomsListView');
+			const rid = result.room._id;
+			navigation.navigate('RoomView', { rid, name, t: 'd' });
+		}
+	}
+
+	makeVideoCall = () => RocketChat.callJitsi(this.rid)
```

Comment:
change not necessary

Reply:
@djorkaeffalexandre Here, I didn't understand , should I remove `makeVideoCall`?